corehq/apps/accounting/static/accounting/js/enterprise_settings.js

```diff
@@ -22,7 +22,8 @@ hqDefine("accounting/js/enterprise_settings", [
             domains: options.restricted_domains.join(", "),
             email: options.accounts_email,
         };
-        self.restrictSignupHelp = _.template(gettext("Do not allow new users to sign up on commcarehq.org." +
+        self.restrictSignupHelp = _.template(gettext("Do not allow new users to sign up on commcarehq.org. " +
+            "This may take up to an hour to take effect. " +
```

Comment:
Is this true or just a warning to be safe?

Reply:
It's true: https://github.com/dimagi/commcare-hq/pull/21403/files#diff-0cd6898a4a91ed249c85e07a44019510R460